utils/python/CIME/test_utils.py

```diff
@@ -47,6 +47,6 @@ def get_tests_from_xml(xml_machine=None,xml_category=None,xml_compiler=None, xml
                                                          testmod=None if "testmods" not in test else test["testmods"])
             logger.debug("Adding test %s with compiler %s"%(test["name"], test["compiler"]))
         listoftests += newtests
-        logger.info("Found %d tests"% len(listoftests))
+        logger.debug("Found %d tests"% len(listoftests))
```

Comment:
I changed this logging level from info to debug to avoid cluttering the output. Personally, I didn't find this output very useful without the other debug output in this function, so it seemed safe to suppress. But if you feel it should be kept in, then I can figure out some other way to suppress the output when this is called from query_testlists.